build.gradle

```diff
@@ -374,6 +374,7 @@ dependencies {
   implementation group: 'com.azure', name: 'azure-messaging-servicebus', version: '7.0.2'
   implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.6'
   implementation group: 'org.springframework', name: 'spring-core', version: '5.3.16'
+ 	implementation group: 'org.springframework', name: 'spring-beans', version: '5.3.18'
```

Comment:
can we remove the extra space for indentation?

Reply:
Ooo that's quite a big request...

Reply:
Addressed here: https://github.com/hmcts/rd-caseworker-ref-api/pull/470/commits/5cde50de91da8036048d4d11343a759580ff6e22 :)